### PR TITLE
Add feb 2018 tech day email and map

### PR DESCRIPTION
**Pull request by RealOrangeOne**

### CODE REVIEW

File: SB2018/2018-01-31.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+Hi all,
+
+Just a reminder and some information on the tech day we're running on Saturday (3rd feb)!
+
+Tech days are a great opportunity for you to get robotics advice and help with technical issues from our volunteers. We may also have a tour of some university facilities as well.
+
+## Location
+The tech day will be held in [Building 25](http://data.southampton.ac.uk/building/25.html). There's a campus map with parking labelled attached.
```
Comment (Adimote) [inline SB2018/2018-01-31.md]: "map with parking labelled attached" doesn't read well, maybe "map attached which shows the recommended parking location"

**Review by Adimote — APPROVED** [review]
LGTM, one comment

File: SB2018/2018-01-31.md (hunk 1 above)
Comment (RealOrangeOne) [reply SB2018/2018-01-31.md]: Yea, i didn't really like that when writing it, but that's much nicer! Will update

File: SB2018/2018-01-31.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+> Southampton\
+> Hampshire\
+> SO17 1BJ
+
+There's a map attached which shows the recommended parking location.
+
+The day will run from 10:00am until 5:00pm, however feel free to come and go as you please.
+
+## Food
+Food on campus is scarse, so we recommend either ordering in food, or bringing something with you. There are a few corner-shops located nearby however.
```
Comment (kierdavis) [inline SB2018/2018-01-31.md]: scarse -> scarce

File: SB2018/2018-01-31.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+to: All Teams
+subject: Tech day reminder - Saturday 3rd Feb
+---
+
+Hi all,
+
+Just a reminder and some information on the tech day we're running on Saturday (3rd feb)!
```
Comment (PeterJCLaw) [inline SB2018/2018-01-31.md]: feb -> Feb

File: SB2018/2018-01-31.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+## Location
+The tech day will be held in [Building 25](http://data.southampton.ac.uk/building/25.html) at the University of Southampton's Highfield Campus:
+
+> University Road\
+> Southampton\
+> Hampshire\
+> SO17 1BJ
+
+There's a map attached which shows the recommended parking location.
```
Comment (PeterJCLaw) [inline SB2018/2018-01-31.md:20]: Maybe add that we'll have signage from that car-park to Building 25 (only if we actually will of course).

Comment (RealOrangeOne) [reply SB2018/2018-01-31.md:20]: We _should_ have signs for this, but I don't think we need to explicitly tell them they exist. They can just appear to help them out

**Review by PeterJCLaw — APPROVED** [review]

**Review by kierdavis — APPROVED** [review]

**Pull request merged**